### ISSUE TITLE
Overriding depreciation values isn't working

### ISSUE THREAD

**Issue by arnaud-leroy** [calliope-project/calliope#64]
Hello,

I try to overwrite the plant_life and monetary.interest for a technology but it isn't changing the cost-result for that technology.

locations.yaml:
```
locations:
    L1:
        techs: [
         'supply_power',
         'demand_power',
         'unmet_demand_power'
         ]
        override:
            demand_power.constraints.r: -10
            supply_power.depreciation.plant_life: 4
            supply_power.depreciation.interest.monetary: 0.5
```
techs.yaml:
```
techs:
    supply_power:
        name: 'Electricity import'
        parent: supply
        carrier: power
        constraints:
            r: inf
            e_cap.max: inf
        costs:
            monetary:
                e_cap: 10
                om_fuel: 1 # €/MWh electricity price
        depreciation:
            plant_life: 20  # Lifetime of a plant (years)

    demand_power:
        name: 'Electrical demand'
        parent: demand
        carrier: power

    unmet_demand_power:
        name: 'Unmet electrical demand'
        parent: unmet_demand
        carrier: power
```

**Comment by brynpickering** [comment]
I have noticed that the calculation of the deprecation rate is location agnostic [here](https://github.com/calliope-project/calliope/blob/2a99495abc770a9038b0f2de3d22cb9e70a9cfa2/calliope/utils.py#L539-L551): 
```
def depreciation_getter(option_getter_func):
    def get_depreciation_rate(y, k):
        interest = option_getter_func(
            y + '.depreciation.interest.' + k,
            default=y + '.depreciation.interest.default')
        plant_life = option_getter_func(y + '.depreciation.plant_life')
        if interest == 0:
            dep = 1 / plant_life
        else:
            dep = ((interest * (1 + interest) ** plant_life)
                   / (((1 + interest) ** plant_life) - 1))
        return dep
    return get_depreciation_rate
```
I'll update it for the 0.5.3 release. For now you should be able to change these settings at a technology level, just without the location-specific override.

**Comment by arnaud-leroy** [comment]
Thanks!
Yes, Putting it for each technology is working.

**Comment by brynpickering** [comment]
Fixed in 77a0930
